app/views/layouts/angular/_per_provider_settings.html.haml

```diff
@@ -0,0 +1,25 @@
+#settings_tabs
+  %h3
+    = _("Settings")
+  %ul.nav.nav-tabs
+    = miq_tab_header('proxy_settings', nil, {'ng-click' => "changeAuthTab('proxy_settings')",
+                                             'ng-if'    => "emsOptionsModel.provider_options.hasOwnProperty('proxy_settings')"}) do
+      %i{"error-on-tab" => "hawkular", :style => "color:#cc0000"}
```

Comment:
just wondering, is this file suppose to be generic? if thats the case, why do you have ```hawkular``` here?

Reply:
Thanks, It was a mistake, I changed it to "proxy_settings" and "advanced_settings" below.